File: labs_java/lab4/SmartHome.java
public class SmartHome {
    private boolean lightsOn;
    private boolean heatingOn;
    private boolean securityOn;

    public boolean isLightsOn(String house) {
        return lightsOn;
    }

    public void setLightsOn(String house, boolean lightsOn) {
        this.lightsOn = lightsOn;
    }

    public boolean isHeatingOn(String house) {
        return heatingOn;
    }

    public void setHeatingOn(String house, boolean heatingOn) {
        this.heatingOn = heatingOn;
    }

    public boolean isSecurityOn(String house) {
        return securityOn;
    }

    public void setSecurityOn(String house, boolean securityOn) {
        this.securityOn = securityOn;
    }
}

File: labs_python/lab4/SmartHome.py
class SmartHome:
    def __init__(self):
        self.lights_on = False
        self.heating_on = False
        self.security_on = False

    def is_lights_on(self):
        return self.lights_on

    def set_lights_on(self, state):
        self.lights_on = state

    def is_heating_on(self):
        return self.heating_on

    def set_heating_on(self, state):
        self.heating_on = state

    def is_security_on(self):
        return self.security_on

    def set_security_on(self, state):
        self.security_on = state
